apps/sms/views/inbox/style/inbox.css

```diff
@@ -122,6 +126,12 @@
   background: url("images/draft.png") no-repeat left center / 1.2rem;
   visibility: visible;
 }
+html[dir="ltr"] [data-type="list"] .draft .icon-draft {
+  background-position: left center;
+}
+html[dir="rtl"] [data-type="list"] .draft .icon-draft {
+  background-position: right center;
+}
```

Comment:
wondering it we can have only one set of rules for both .icon-draft and .icon-unread by using simply .icon.